packages/gatsby-source-wikipedia/src/fetch.js

```diff
@@ -75,8 +75,11 @@ const search = ({ query, limit }) =>
       })
     )
 
-const getArticle = name =>
-  fetch(`https://en.m.wikipedia.org/wiki/${name}?action=render`)
+/**
+ * You can get wikipedia articles in different languages
+ */
+const getArticle = (name, lang = 'en') =>
+  fetch(`https://${lang}.m.wikipedia.org/wiki/${name}?action=render`)
     .then(res => res.text())
     .then(pageContent =>
       pageContent.replace(/\/\/en\.wikipedia\.org\/wiki\//g, `/wiki/`)
```

Comment:
This `en` here probably also needs to be handled?

Reply:
I don't really see what this replace is doing. I could not add the `lang` variable in that regex. But maybe if I just write it as a string.